interfaces/Ids.h

```diff
@@ -211,7 +211,20 @@ namespace Exchange {
         ID_LANGUAGETAG_NOTIFICATION,
           
         ID_EXTERNAL_DERIVED,
-        ID_BRIGHTNESS
+        ID_BRIGHTNESS,
+
+        //OCDM
+        ID_ACCESSOROCDM,
+        ID_ACCESSOROCDM_NOTIFICATION,
+        ID_SESSION,
+        ID_SESSION_CALLBACK,
+        ID_SESSION_EXTENSION,
+
+        //Butler
+        ID_GID_BUTLER_GROUPROUP,
```

Comment:
Eh, did you compile this? At least it looks different then the one used in the IButler.h file :)